internal/server/statvar/statvar_match.go

```diff
@@ -16,39 +16,293 @@ package statvar
 
 import (
 	"context"
+	"database/sql"
+	"encoding/json"
 	"fmt"
+	"hash/fnv"
 	"math"
 	"sort"
 	"strings"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3"
 
-	"github.com/blevesearch/bleve/v2"
-	"github.com/blevesearch/bleve/v2/search"
 	pb "github.com/datacommonsorg/mixer/internal/proto"
 	"github.com/datacommonsorg/mixer/internal/server/resource"
 	"github.com/datacommonsorg/mixer/internal/store"
+	"github.com/datacommonsorg/mixer/internal/util"
 )
 
-func buildExplanationString(in *search.Explanation, sb *strings.Builder, level int) {
-	sb.WriteString(fmt.Sprintf("%.2f => %s\n", in.Value, in.Message))
-	if len(in.Children) > 0 {
-		for _, cchild := range in.Children {
-			for i := 0; i < level+1; i++ {
-				sb.WriteString(" ")
+const defaultLimit = 5
+
+const sqlCreateTables = `
```

Comment:
Can you add some comments for these commands?

Reply:
Maybe some higher level comments on how the sqlite search does?

Reply:
Done.